### PR TITLE
Add function to calculate bounding box

### LINKED ISSUE #148: Functionality to get bounding box of geometry
`> AG.envelope(geom)` 

`GDAL.OGREnvelope(12.445608139038143, 12.458451271057186, 41.90011215209961, 41.90755081176758)`

Instead of the OGREnvelope, it would be beneficial if I can get a POLYGON type bounding box from these coordinates.

Suggestions:

`AG.boundbox2d(geom)` which will give me a POLYGON shape

I would like to work on this feature.

### PR DESCRIPTION
**Pull request by zerefwayne**
closes #148 

API: `AG.boundingbox(geom::AbstractGeometry)` returns Clockwise Polygon

- [x] Add boundingbox function
- [x] Add test for function

### CODE REVIEW

**Comment by zerefwayne** [comment]
@yeesian The coverage decreased because of the addition of 1 more function. I have done complete testing.

**Comment by zerefwayne** [comment]
ping @yeesian

File: src/ogr/geometry.jl [hunk 1]
```diff
@@ -214,6 +214,18 @@ function envelope3d(geom::AbstractGeometry)
...
 
+"""
+    boundingbox(geom::AbstractGeometry)
+
+Returns a bounding box polygon (CW) built from envelope coordinates
+"""
+function boundingbox(geom::AbstractGeometry)
+    coordinates = envelope(geom)
+    MinX, MaxX, MinY, MaxY = coordinates.MinX, coordinates.MaxX, coordinates.MinY, coordinates.MaxY
+    # creates a CW closed ring polygon
```
Comment (yeesian) [inline src/ogr/geometry.jl:225]: nit: is this comment necessary?

**Review by yeesian — APPROVED** [review]
This PR is great, thank you!

(Sorry for the slow review from me -- it's been really intense for me at work lately.)

File: src/ogr/geometry.jl (hunk 1 above)
Comment (zerefwayne) [reply src/ogr/geometry.jl:225]: It's good to know what orientation of a ring I'll recieve if someone's confused.

**Pull request merged**